gradle.properties

```diff
@@ -3,7 +3,7 @@ org.gradle.caching=true
 org.gradle.parallel=true
 
 # Project properties --------------------------
-version=0.9.1
+version=0.9.2
```

Comment:
No need to change it. 
I will decide a release version. Current `0.9.1-SNAPSHOT` is fine